### ISSUE TITLE
Legend of attached files can't be translated

### ISSUE THREAD

**Issue by CynthiaBorotPNV** [GeotrekCE/Geotrek-admin#1674]
There no way to translate the legend/alternative text of attached files. Could you add this field to translation mecanism ? 

Thanks

**Comment by camillemonchicourt** [comment]
Yes would be good. But as attached files are also used for MAINTENANCE modules, it may can be a problem.
